src/templates/home-page/Hero/index.tsx

```diff
@@ -56,6 +56,7 @@ const Hero: FC<{
                       {...heroButtonProps}
                       variant={index === 0 ? "special" : "outline"}
                       href={_.url as ExternalLinkHref}
+                      data-ph-capture-attribute-button-name={`${_.label} (${window.location.href})`}
```

Comment:
I believe we shouldn't use `window.location` because Gatsby generates pages on the server side during the build. Please check this out to see how to get the url in gatsby: https://www.gatsbyjs.com/docs/location-data-from-props/#getting-the-absolute-url-of-a-page

Reply:
I've change it a little bit and I've added `posthogLabel` to the md files for specific buttons. You can see the changes here c32c39a47c207c9aaea6aaf9a3f14063e383a28e